Styling/schema/style.meta.schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema" : "http://json-schema.org/draft-03/schema",
+    "title" : "meta",
+    "type" : "object",
+    "description" : "A series of property names and the expression to evaluate for the value of each property.",
+    "properties" : {
+    },
+    "additionalProperties" : {
+        "$ref" : "style.metaProperty.schema.json"
```

Comment:
Ah, I see.  Can you make the description more precise since we have only defined boolean and color expressions and this could also be a constant string, expression returning a number, etc.